src/pages/SignUpForm/SignUpForm.js

```diff
@@ -13,31 +52,66 @@ const SignUpForm = () => {
         <div className="signUpFormInputContainerBox">
           <div className="signUpFormInputBox">
             <p className="signUpFormInputName">아이디(이메일 주소)</p>
-            <input className="signUpFormInputText" type="text" />
+            <input
+              className="signUpFormInputText"
+              type="text"
+              onChange={handleLogin}
+              name="email"
+              value={userInfo.email}
+              placeholder="email 형식"
+            />
           </div>
           <div className="signUpFormInputBox">
             <p className="signUpFormInputName">비밀번호</p>
-            <input className="signUpFormInputText" type="password" />
+            <input
+              className="signUpFormInputText"
+              type="password"
+              name="password"
+              onChange={handleLogin}
+              placeholder="비밀번호 8자리 이상"
+            />
           </div>
           <div className="signUpFormInputBox">
             <p className="signUpFormInputName">비밀번호 확인</p>
-            <input className="signUpFormInputText" type="password" />
+            <input
+              className="signUpFormInputText"
+              type="password"
+              onChange={handleLogin}
+              name="checkPassword"
+              placeholder="비밀번호 8자리 이상"
+            />
           </div>
           <div className="signUpFormInputBox">
             <p className="signUpFormInputName">주소</p>
-            <input className="signUpFormInputText" type="text" />
+            <input
+              className="signUpFormInputText"
+              type="text"
+              name="address"
+              onChange={handleLogin}
+            />
           </div>
 
           <div className="signUpFormInputLastBox">
             <p className="signUpFormInputName">이름</p>
-            <input className="signUpFormInputText" type="text" />
+            <input
+              className="signUpFormInputText"
+              type="text"
+              name="name"
+              onChange={handleLogin}
+            />
           </div>
         </div>
         <div className="signUpFormButton">
-          <Link to="/sign-up">
-            <button className="signUpFormBackBtn">이전으로</button>
-          </Link>
-          <button className="signUpFormBtn">가입하기</button>
+          {/* <Link to="/sign-up"> */}
+          <button className="signUpFormBackBtn">이전으로</button>
```

Comment:
```suggestion
          <button className="signUpFormBackBtn" onClick={()=>navigate (-1)}>이전으로</button>
```
`const navigate = useNavigate();`